ironfish-cli/src/commands/wallet/multisig/participant/create.ts

```diff
@@ -50,4 +69,20 @@ export class MultisigIdentityCreate extends IronfishCommand {
     this.log('Identity:')
     this.log(response.content.identity)
   }
+
+  async createParticipantWithLedger(): Promise<Buffer> {
```

Comment:
getParticipant/ IdentityFromLedger might be more accurate?

Reply:
renamed to `getIdentityFromLedger`